blogposts/2022/release-3.38.md

```diff
@@ -0,0 +1,74 @@
+    url: https://github.com/sourcegraph/sourcegraph/pull/31520
+    category: Search
+---
+
+Sourcegraph 3.38 is now available! For this release, we introduced:
+
+## Code Insights is generally available
+
+[Code Insights](https://docs.sourcegraph.com/code_insights) transforms any search query into customizable, visual dashboards in seconds. Code Insights surfaces high-level information about your codebase, based on both how your code changes over time and its current state. You can learn more about why we built Code Insights in [the announcement from our CEO](https://about.sourcegraph.com/blog/announcing-code-insights/).
+
+Any Sourcegraph instance can create up to two code insights, even without a trial or license. Reach out to your account team to purchase access to the full version.
+
+<div class="container my-4 video-embed embed-responsive embed-responsive-16by9">
+  <iframe
+    class="embed-responsive-item"
+    src="https://www.youtube-nocookie.com/embed/fMCUJQHfbUA?autoplay=0&amp;cc_load_policy=0&amp;start=0&amp;end=0&amp;loop=0&amp;controls=1&amp;modestbranding=1&amp;rel=0"
+    allowfullscreen="true"
+    allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+    frameborder="0"
+    title="Sourcegraph Code Insights demo"
+  ></iframe>
+</div>
+
+## Notebooks are improved with a host of new features
+
+Notebooks have several new features in 3.38 to help your team better understand your codebase:
+
+- You can now copy any existing Notebook to your own collection of Notebooks.
+- We've created a new symbol block type. This block lets you find any symbol in a repository, and the block will display that symbol no matter where it moves within a file.
+- Code Intelligence is now available in Notebook blocks, making Notebooks more intuitive to be used for onboarding and learning about your codebase.
+- Embedding Notebooks is here! You can now easily embed Notebooks from any Sourcegraph instance by embedding an iFrame anywhere, and setting `src` to `https://sourcegraph.com/notebooks/embed/<notebook_uuid/>`.
+
+## Improved Code Intelligence performance for large repositories
+
+We're introducing a new backend service, Rockskip, for search-based Code Intelligence, the symbol sidebar, and symbol search. Rockskip was architected with monorepos in mind, and it can index new commits much faster than the existing SQLite backend.
+
+From 3.38 onwards, Rockskip can be turned on for specific repositories and it will make the symbol sidebar and search-based Code Intelligence much faster. If you're interested in turning this on for your repositories, read more in the [docs](https://docs.sourcegraph.com/code_intelligence/explanations/rockskip) or [contact us](mailto:support@sourcegraph.com).
+
+<img class="blog-image" title="Rockskip performance improvement" alt="Comparative chart showing impact of incremental indexing" src="https://storage.googleapis.com/sourcegraph-assets/docs/images/code-intelligence/rockskip-chart.png">
```

Comment:
Also not blocking, but ideally where we have charts/graphics in a post we should have these done according to our brand guidelines (i.e. in our colour palette and with our brand typefaces). We can request help from our design agency via Fabiana if it can't be done in house 😃

Reply:
Thanks @rebeccadee! In this case I'm a bit mixed on the visual as-is, since it doesn't show anything in the product and feels a bit out of place. For now, I am going to remove this one. Will definitely use brand guidelines for any type of charting / custom images going forward!